balboa-kafka/src/main/scala/com/socrata/balboa/kafka/BalboaConsumerGroup.scala

```diff
@@ -0,0 +1,55 @@
+package com.socrata.balboa.kafka
+
+import java.util.concurrent.{Executors, ExecutorService}
+
+import com.socrata.balboa.kafka.consumer.TestConsumer
+import kafka.consumer.{KafkaStream, ConsumerConnector, Consumer, ConsumerConfig}
+
+import scala.collection.Map
+import scala.collection.immutable.HashMap
+
+/**
+ * Created by Michael Hotan, michael.hotan@socrata.com
+ *
+ * ADT that manages the connection with zookeeper nodes and consumption of Kafka Messages.  ADT attempts to utilize a
+ * pure functional
+ *
+ * Note: Most likely won't
+ */
+sealed trait BalboaConsumerGroup
+case class ::(consumer: ConsumerConnector, topic: String) extends BalboaConsumerGroup
+
+/**
+ * `BalboaKafka` Companion object
```

Comment:
This is not a companion object to BalboaKafka; and anyways... if they are in the same file you don't need to mention it.